src/borg/archiver/create_cmd.py

```diff
@@ -370,6 +371,10 @@ def _process_any(self, *, path, parent_fd, name, st, fso, cache, read_special, d
                     self.print_warning("Unknown file type: %s", path)
                     return
             except (BackupError, BackupOSError) as err:
+                if err is BackupOSError:
+                    if err.errno == errno.EACCES or err.errno == errno.EPERM:
+                        # Do not try again, permission errors can not be fixed by retrying.
```

Comment:
in case we add other errors later, maybe rather say "such errors" instead "permission errors".